src/python/pants/backend/python/tasks2/pytest_run.py

```diff
@@ -80,6 +80,8 @@ def register_options(cls, register):
                   'create a new chroot, which will be much slower, but more correct, as the '
                   'isolation verifies that all dependencies are correctly declared.')
     register('--junit-xml-dir', metavar='<DIR>',
+             removal_version='1.5.0.dev0',
+             removal_hint='Unused. We always use JUnit XML results files.',
```

Comment:
seems like this would still be generally useful for emitting a copy of the junit xml results to a workdir-external path? from a quick look, it appears we actually use this for that purpose in our CI.